cmd/push.go

```diff
@@ -61,6 +61,10 @@ var pushCmd = &cobra.Command{
 		case "local", "binary":
 			// use value of '--dir' as source if it was used
 			if len(componentLocal) != 0 {
+				if sourceType == "binary" {
+					fmt.Println("unable to push local directory:" + componentLocal + " to component " + componentName + " that uses binary " + sourcePath)
```

Comment:
Please use `fmt.Printf`. The message should be a sentence starting with the capital letter and ending with '.',

Reply:
@kadel Done